task6/script.js

```diff
@@ -0,0 +1,293 @@
+        },
+
+        {
+            id: '12',
+            description: 'Более 76 тыс. человек во всем мире уже излечились от заболевания, спровоцированного новым коронавирусом, тогда как количество смертей превысило 6,4 тыс.',
+            createdAt: new Date('2019-03-17T23:00:00'),
+            author: 'Иванов Иван'
+        },
+
+        {
+            id: '13',
+            description: 'Более 76 тыс. человек во всем мире уже излечились от заболевания, спровоцированного новым коронавирусом, тогда как количество смертей превысило 6,4 тыс.',
+            createdAt: new Date('2019-03-17T23:00:00'),
+            author: 'Иванов Иван'
+        },
+
+        {
+            id: '14',
+            description: 'Более 76 тыс. человек во всем мире уже излечились от заболевания, спровоцированного новым коронавирусом, тогда как количество смертей превысило 6,4 тыс.',
+            createdAt: new Date('2019-01-17T23:00:00'),
+            author: 'Иванов Иван'
+        },
+
+        {
+            id: '15',
+            description: 'Более 76 тыс. человек во всем мире уже излечились от заболевания, спровоцированного новым коронавирусом, тогда как количество смертей превысило 6,4 тыс.',
+            createdAt: new Date('2019-03-17T23:00:00'),
+            author: 'Иванов Иван'
+        },
+
+        {
+            id: '16',
+            description: 'Более 76 тыс. человек во всем мире уже излечились от заболевания, спровоцированного новым коронавирусом, тогда как количество смертей превысило 6,4 тыс.',
+            createdAt: new Date('2019-03-17T23:00:00'),
+            author: 'Иванов Иван'
+        },
+
+        {
+            id: '17',
+            description: 'Более 76 тыс. человек во всем мире уже излечились от заболевания, спровоцированного новым коронавирусом, тогда как количество смертей превысило 6,4 тыс.',
+            createdAt: new Date('2019-03-17T23:00:00'),
+            author: 'Иванов Иван'
+        },
+
+        {
+            id: '18',
+            description: 'Более 76 тыс. человек во всем мире уже излечились от заболевания, спровоцированного новым коронавирусом, тогда как количество смертей превысило 6,4 тыс.',
+            createdAt: new Date('2019-03-17T23:00:00'),
+            author: 'Иванов Иван'
+        },
+
+        {
+            id: '19',
+            description: 'Более 76 тыс. человек во всем мире уже излечились от заболевания, спровоцированного новым коронавирусом, тогда как количество смертей превысило 6,4 тыс.',
+            createdAt: new Date('2019-03-17T23:00:00'),
+            author: 'Николай'
+        },
+
+        {
+            id: '20',
+            description: 'Более 76 тыс. человек во всем мире уже излечились от заболевания, спровоцированного новым коронавирусом, тогда как количество смертей превысило 6,4 тыс.',
+            createdAt: new Date('2021-03-17T23:00:00'),
+            author: 'Николай'
+        },
+
+    ];
+
+    
+    function getPost(id) {
+        for (let i = 0; i < posts.length ; i++) {
```

Comment:
Try to avoid manual iteration over array and prefer default array's methoths, here `find` should be used https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Array/find